app/content/tiles/jadeCityDark.ts

```diff
@@ -20,14 +22,12 @@ import {
 } from 'app/content/bitMasks';
 
 
-import {
-    southernWallBehavior,
-    topRightWall,
-    topLeftWall,
-} from 'app/content/tiles/constants';
+const topLeftWall: TileBehaviors = { defaultLayer: 'field', solidMap: BITMAP_TOP_LEFT, isSouthernWall: true, isGround: false, linkedOffset: -159};
```

Comment:
Please use the same `const linkedTopLeftWall: TileBehaviors = {...topLeftWall, ` that I recommended in another file.